pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -60,7 +61,7 @@ func (l *listUsersQuery) ListUsers(
 	foundUsersCh := make(chan *openfgav1.User, 1)
 	expandErrCh := make(chan error, 1)
 
-	var foundUsers []*openfgav1.User
+	foundUsers := make([]*openfgav1.User, 0)
```

Comment:
curious why you don't use   
```
var foundUsers []*openfgav1.User
```
instead

Reply:
Because don't want to return nil if no users found.